tests/mcs_benchmark_data_test/assertions.py

```diff
@@ -1,13 +1,14 @@
-from mcs_benchmark_data.path import TEST_DATA_DIR_PATH
+from mcs_benchmark_data.path import DATA_DIR_PATH
 from rdflib import Graph
+from pathlib import Path
 import bz2
 
 from mcs_benchmark_data.models.submission_sample import SubmissionSample
 
 
-def assert_valid_rdf_loaded_test(pipeline_id: str):
+def assert_valid_rdf_loaded_test(pipeline_id: str, data_dir_path: Path = DATA_DIR_PATH):

```

Comment:
If this is suffixed _test it should always be using TEST_DATA_DIR_PATH, so you don't need data_dir_path as a parameter.

Please fix it in the next branch. I'm going to approve and merge this one.